server/db/recreate-schema.sql

```diff
@@ -34,5 +34,6 @@ CREATE TABLE documents (
   submission_date  DATE NOT NULL,
   due_date         DATE NOT NULL,
   owner_id         uuid REFERENCES users(id) NOT NULL,
-  format           format_types
+  format           format_types,
+  content          TEXT NOT NULL
```

Comment:
Now that the content column is defined as not null, the recreate-db command doesn't work anymore when creating data in the database.